app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -46,8 +46,6 @@ Your training provider will aim to make a decision on whether to make an offer w
 
 # Need help with your teacher training application?
 
-Call [Get into Teaching](https://getintoteaching.education.gov.uk/) on Freephone 0800 389 2500 or chat to an adviser using their [online chat service](https://getintoteaching.education.gov.uk/lp/live-chat) between 8am and 8pm, Monday to Friday.
+Contact the team on [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
 
-# Give feedback or report a problem
-
-Please contact the team on [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk)
+We typically respond within one working day for urgent queries. It can take up to 5 working days for less urgent queries. Unfortunately, we can’t provide support over the weekend.
```

Comment:
```suggestion
We typically respond within one working day for urgent queries. It can take up to 5 working days for less urgent queries. We don’t provide support over the weekend.
```

It's not that we can't, it's that we choose not to.